File: core-java-program/gcr-codebase/programming-elements/CylinderVol.java
import java.util.Scanner;
public class CylinderVol{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter Radius: ");
		double radius = sc.nextDouble();
		System.out.print("Enter Height: ");
		double height = sc.nextDouble();
		double volume = 3.14*radius*radius*height;
		System.out.println("Volume is: " + volume);
	}
}

File: core-csharp-program/gcr-codebase/programming-elements/CylinderVol.cs
using System;

public class CylinderVol
{
    public static void Main(string[] args)
    {
        Console.Write("Enter Radius: ");
        double radius = Convert.ToDouble(Console.ReadLine());

        Console.Write("Enter Height: ");
        double height = Convert.ToDouble(Console.ReadLine());

        double volume = 3.14 * radius * radius * height;
        Console.WriteLine("Volume is: " + volume);
    }
}
